packages/medusa/src/services/order-edit.ts

```diff
@@ -213,7 +213,7 @@ export default class OrderEditService extends TransactionBaseService {
 
   async create(
     data: CreateOrderEditInput,
-    context: { loggedInUserId: string }
+    context: { createdBy: string }
```

Comment:
todo: `createdBy` should not be required
```suggestion
    context: { createdBy?: string } = {}
```

Reply:
`created_by` is required at the database level. Do you think we should make it nullable?

Reply:
Ah thought it was... but yes would definitely suggest that we make that nullable

Reply:
We can skip this for now though